settingshelper.py

```diff
@@ -101,10 +115,7 @@ def make_couchdb_tuples(self):
         return [self._make_couchdb_tuple(row) for row in self.couchdb_apps]
 
     def _format_db_uri(self, db_uri):
-        if self.is_test:
-            return '{}_test'.format(db_uri)
-        else:
-            return db_uri
+        return db_uri
```

Comment:
@dannyroberts I removed the `is_test` flag here. (I think it was not used anyway.) It is no longer needed since the database name is set correctly from the very beginning now instead of later during test runner setup.